### PR TITLE
Can't move past wizard screen

### PR DESCRIPTION
**Pull request by jerith**
On staging - on the wizard screen: http://qa-vumi.za.prk-host.net/wizard/create/

I provide a conversation name and description.
I pick 'dialogue' as the conversation type
I select "set up a new channel" and pick a channel
I click the 'save' button
Instead of being directed to the survey builder, I remain on the wizard screen: http://qa-vumi.za.prk-host.net/wizard/create/ and get a blank page

screen shot 2013-08-06 at 2 00 56 pm

screen shot 2013-08-06 at 2 01 50 pm

### CODE REVIEW

**Comment by hodgestar** [comment]
Can you retest quickly? @smn fixed #557 this morning and I've just deployed.

File: go/wizard/views.py [hunk 1]
```diff
@@ -68,6 +68,8 @@ def post(self, request):
             # TODO: Better validation.
             logger.info("Validation failed: %s" % (
                 [frm.errors for frm in forms_to_validate],))
+            print "Validation failed: %s" % (
+                [frm.errors for frm in forms_to_validate],)
             return self._render(request, wizard_form=wiz_form,
```
Comment (justinvdm) [inline go/wizard/views.py:70]: Stray debug statement?

File: go/wizard/views.py [hunk 2]
```diff
@@ -94,7 +93,7 @@ def post(self, request):
                 'show', conversation_key=conv.key))
 
     def _handle_new_channel(self, request, chan_data, keyword, conv):
-        channel = chan_data['channel'].split(':')
+        channel = tuple(chan_data['channel'].split(':'))
```
Comment (hodgestar) [inline go/wizard/views.py]: This should probably be a `split(':', 1)` for safety?

**Comment by hodgestar** [comment]
:+1:

**Pull request merged**